identity-iota/src/tangle/compression_brotli.rs

```diff
@@ -0,0 +1,44 @@
+// Copyright 2020-2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::error::Error;
+use crate::error::Error::CompressionError;
+use std::io::prelude::*;
+
+pub(crate) fn compress_brotli<T: AsRef<[u8]>>(input: T) -> Result<Vec<u8>, Error> {
+  let mut result = Vec::new();
+  let mut compressor = brotli::CompressorReader::new(input.as_ref(), 4096, 5, 22);
+  compressor
+    .read_to_end(&mut result)
+    .map_err(|_| Error::CompressionError)?;
+  Ok(result)
+}
+
+pub(crate) fn decompress_brotli<T: AsRef<[u8]> + ?Sized>(input: &T) -> Result<String, Error> {
+  let mut decompressor = brotli::Decompressor::new(input.as_ref(), 4096 /* buffer size */);
```

Comment:
Please declare constants for magic numbers like buffer sizes at the top of the file.

E.g.
```rust
const BUFFER_SIZE: usize = 4096;
```